profile/README.md

```diff
@@ -28,20 +28,20 @@ This repository stores all the code for Polygon Miden. Everything is open source
 | [Examples](https://github.com/0xPolygonMiden/examples)        | Examples of using the Miden VM and Miden assembly. |
 
 ## Status
-At the moment you can use the Miden VM v0.6 on our [playground](https://0xpolygonmiden.github.io/examples/) and locally. Check out the Miden VM repository or our [Miden VM docs](https://wiki.polygon.technology/docs/miden/intro/main) for more detailed instructions.
-    
-As soon as we have the Miden Client or Miden Node, we will make it available. Follow us on Twitter to get the latest updates.
+We have a [public testnet](https://docs.polygon.technology/miden/miden-base/introduction/get-started/create-account-use-faucet/), and you can interact with our Miden node using the Miden client. Currently, you can only use predefined smart contracts and node scripts. As soon as we have our compiler, builders can start writing their own contracts.
+
+You can also use the Miden VM v0.9 locally and on our [playground](https://0xpolygonmiden.github.io/examples/). For more detailed information, check out the Miden VM repository or our [Miden VM docs](https://wiki.polygon.technology/docs/miden/intro/main).
     
-In case you want to use parts of Polygon Miden - like the Miden VM or AirScript - please reach out to [us](dschmid@polygon.technology).
+If you want to use parts of Polygon Miden - like the Miden VM or AirScript - please contact [us](dschmid@polygon.technology).
```

Comment:
A couple of comments here:

- I would get rid of the first-person language. For example, I'd say something like "Miden testnet is already up and running ..." instead of "We have a public testnet".
- We should probably link to `testnet.miden.io` from here (I know the link is in the tutorial, but I'm thinking a direct link might be good as well). Btw, I'm thinking we should add some links to the page at `testnet.miden.io` (e.g., link to the docs, link to the block explorer).